### ISSUE TITLE
Failed to execute script peld

### ISSUE THREAD

**Issue by Lucifirius** [ArtificialQualia/PyEveLiveDPS#5]
I get this error when I launch the .exe file. I have redownloaded multiple times.

**Comment by ArtificialQualia** [comment]
@Lucifirius Are you using the latest version? (v1.0.4)

If you are, can you download this version and copy+paste the error you get in the console:

https://github.com/ArtificialQualia/PyEveLiveDPS/releases/download/v1.0-beta/peld_v1.0.5-debug.zip

**Comment by Lucifirius** [comment]
Yes, latest version. 

```
Traceback (most recent call last):
  File "peld.py", line 20, in <module>
  File "peld.py", line 16, in __init__
  File "window.py", line 136, in __init__
  File "logreader.py", line 54, in __init__
  File "logreader.py", line 80, in addLog
  File "logreader.py", line 134, in __init__
  File "c:\python35\lib\encodings\cp1252.py", line 23, in decode
UnicodeDecodeError: 'charmap' codec can't decode byte 0x8f in position 20689: character maps to <undefined>
Press any key to close console
```

This version runs though, and gives the DPS window.

**Comment by ArtificialQualia** [comment]
Thanks for the error data.

It appears you have a corrupted log file.

I'll add some error handling to try to handle this case without user intervention and no interruption in service.  I'll probably have a working release for you tonight.

**Comment by Lucifirius** [comment]
Cool, thanks!

**Comment by ArtificialQualia** [comment]
Turns out the wrong encoding was being used to read the log files, but it only caused errors in rare cases.

I've fixed your issue in the new v1.0.5 release (along with a few other things I've been working on)

You can download it from the releases page, or directly here:
https://github.com/ArtificialQualia/PyEveLiveDPS/releases/download/v1.0.5/peld_v1.0.5.zip

If you have any further issues let me know.

**Comment by Lucifirius** [comment]
Works like a charm! Thanks!
